### PR TITLE
Qa/followup code snippets

### PR DESCRIPTION
**Pull request by msdolbey**
## Jira Ticket

n/a

## Description of changes being made

Making code bracket lang for commands similar/outputs are shell

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4194.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/self-managed/index.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ Before starting, ensure you create a workload cluster as described in [Create a
 
     The cluster lifecycle services on the workload cluster are ready, but the workload cluster configuration is on the bootstrap cluster. The `move` command moves the configuration, which takes the form of Cluster API Custom Resource objects, from the bootstrap to the workload cluster. This process is also called a [Pivot][pivot].
 
-    ```sh
+    ```bash
     dkp move --to-kubeconfig ${CLUSTER_NAME}.conf
     ```
 
-    ```sh
+    ```bash
```
Comment (gflorinskaya) [inline pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/self-managed/index.md:45]:
```suggestion
```sh
```

File: pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md [hunk 2]
```diff
@@ -76,26 +76,26 @@ Tips:
...
     This creates a unique name every time you run it, so use the command with forethought.
 
-    ```sh
+    ```bash
     CLUSTER_NAME=$(whoami)-aws-cluster-$(LC_CTYPE=C tr -dc 'a-z0-9' </dev/urandom | fold -w 5 | head -n1)
     echo $CLUSTER_NAME
     ```
 
-    ```text
+    ```bash
```
Comment (gflorinskaya) [inline pages/dkp/konvoy/2.2/choose-infrastructure/eks/quick-start/index.md:84]:
```suggestion
```sh
```

**Comment by msdolbey** [comment]
Thanks @gflorinskaya ! merged those suggestions.

**Review by mwrightd2 — APPROVED** [review]
These are just the new bash wrapper changes right? Approved.

**Review by gracedo — APPROVED** [review]

**Pull request merged**